pandas/tseries/tools.py

```diff
@@ -483,6 +483,49 @@ def parse_time_string(arg, freq=None, dayfirst=None, yearfirst=None):
 normalize_date = tslib.normalize_date
 
 
+# Used in between_time string parsing
+_time_formats = ["%H:%M", "%H%M", "%I:%M%p", "%I%M%p", 
+                 "%H:%M:%S", "%H%M%S", "%I:%M:%S%p", "%I%M%S%p"]
+
+def add_time_format(time_format):
+    _time_formats.append(time_format)
+
+def to_time(arg, format=None):
+    """
+    Parse time strings to time objects using fixed strptime formats.
+
+    Parameters
+    ----------
+    arg : compat.string_types
+    format : str, list-like, default None
+        Format used to convert arg into a time object.  If None, default formats
+        are used.  To add an additional format use 
```

Comment:
More for backwards compatibility.  Our formats are a subset of dateutil's so want to ensure that the code still works for those that may have used a different format historically.